code/game/dna/genes/goon_powers.dm

```diff
@@ -657,25 +657,24 @@
 	action_icon_state = "superfart"
 
 /obj/effect/proc_holder/spell/aoe_turf/superfart/invocation(mob/user = usr)
-	invocation = "<span class='warning'><b>[user]</b> hunches down and grits their teeth!</span>"
-	invocation_emote_self = invocation
+	invocation = "<span class='warning'>[user] hunches down and grits their teeth!</span>"
+	invocation_emote_self = "<span class='warning'>You hunch down and grit your teeth!</span>"
 	..(user)
 
-/obj/effect/proc_holder/spell/aoe_turf/superfart/cast(list/targets)
-	var/UT = get_turf(usr)
-
-	if(do_after(usr, 30, target = usr))
+/obj/effect/proc_holder/spell/aoe_turf/superfart/cast(list/targets, mob/user)
+	var/UT = get_turf(user)
+	if(do_after(user, 30, target = user))
 		playsound(UT, 'sound/goonstation/effects/superfart.ogg', 50, 0)
-		usr.visible_message("<span class='warning'><b>[usr]</b> unleashes a [pick("tremendous","gigantic","colossal")] fart!</span>", "<span class='warning'>You hear a [pick("tremendous","gigantic","colossal")] fart.</span>")
+		user.visible_message("<span class='warning'><b>[user]</b> unleashes a [pick("tremendous","gigantic","colossal")] fart!</span>", "<span class='warning'>You hear a [pick("tremendous","gigantic","colossal")] fart.</span>")
```

Comment:
Seems silly but, with the two separate picks, it may show `colossal` to the user, but everyone else thinks its `tremendous`.